ceph/ceph_admin/bootstrap.py

```diff
@@ -179,9 +179,17 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
         args = config.get("args")
         custom_repo = args.pop("custom_repo", None)
         custom_image = args.pop("custom_image", True)
+        build = config.get("build")
+
+        if build == "released":
```

Comment:
As per our discussion, I think we thought of not using custom_image parameter from test case config.
``` custom_image = False
if build == "released" or _custom_repo.lower() ==  "cdn":
	custom_repo = "cdn"
	custom_image = False
else if _custom_repo != "cdn":
	custom_repo = base_url
	custom_image = container_image 
```

Reply:
Yes @sunilkumarn417 later i missed out some scenarios and it's updated accordingly now
Thanks Please review.